packages/core/useTextareaAutosize/index.md

```diff
@@ -6,8 +6,25 @@ category: Browser
 
 Automatically update the height of a textarea depending on the content.
 
+::: info
+Don't forget to reset the scrollbar styles for the textarea element to avoid incorrect height values for large amounts of text.
```

Comment:
```suggestion
It's recommended to reset the scrollbar styles for the textarea element to avoid incorrect height values for large amounts of text.
```